.gitignore

```diff
@@ -70,16 +70,13 @@
 !**/vendor/bundle/ruby/*/gems/activesupport-*/lib/active_support/core_ext/hash/except.rb
 !**/vendor/bundle/ruby/*/gems/activesupport-*/lib/active_support/core_ext/hash/keys.rb
 !**/vendor/bundle/ruby/*/gems/activesupport-*/lib/active_support/core_ext/hash/slice.rb
-!**/vendor/bundle/ruby/*/gems/activesupport-*/lib/active_support/core_ext/object/blank.rb
 !**/vendor/bundle/ruby/*/gems/activesupport-*/lib/active_support/core_ext/object/deep_dup.rb
 !**/vendor/bundle/ruby/*/gems/activesupport-*/lib/active_support/core_ext/object/duplicable.rb
 !**/vendor/bundle/ruby/*/gems/activesupport-*/lib/active_support/core_ext/string/exclude.rb
 !**/vendor/bundle/ruby/*/gems/activesupport-*/lib/active_support/core_ext/string/filters.rb
 !**/vendor/bundle/ruby/*/gems/activesupport-*/lib/active_support/core_ext/string/indent.rb
 
 # Ignore partially included gems where we don't need all files
-**/vendor/bundle/ruby/*/gems/concurrent-ruby-*/lib/*/*.jar
-**/vendor/bundle/ruby/*/gems/i18n-*/lib/i18n/tests*
```

Comment:
The `i8n` cleanup is unrelated, it should have been included in https://github.com/Homebrew/brew/pull/15311
Let me know if you'd like a cleaner diff, and I can move this to a separate PR.

Reply:
@dduugg Fine with me as-is, thanks!